lib/vagrant.pm

```diff
@@ -33,16 +43,14 @@ sub run_vagrant_cmd {
 
     my $logfile = 'vagrant_cmd.log';
     local $@;
-    my $ret = eval {
-        return script_run("VAGRANT_LOG=DEBUG vagrant $cmd 2> $logfile", %args);
-    };
+    my $ret = script_run("VAGRANT_LOG=DEBUG vagrant $cmd 2> $logfile", %args);
```

Comment:
This is essentially just `assert_script_run` now

Reply:
Except that it uploads the full vagrant debug log.

Reply:
done